test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -100,7 +101,7 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
   )(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        post(urlPathEqualTo(s"$calculationsUrl"))
+        post(urlPathEqualTo(s"${config.mibCalculationsUrl}"))
```

Comment:
This can just be `post(urlPathEqualTo("/declare-commercial-goods/calculations"))`